standard/sections/clause_7_normative_text.adoc

```diff
@@ -77,31 +77,32 @@ The table below provides an overview of the set of properties that may be includ
 
 |``links``
 |**required**
-|Online linkages to data retrieval or additional resources associated with the dataset (see <<_links>>)
+|Online linkages for data retrieval or additional resources associated with the dataset (see <<_links>>)
 
 |===
 
 
 ==== Message size
 
-The WIS2 Notification Message allows for transmission of messages in a compacted or minified manner, and includes the ability to embedding content inline as required (see <<Properties / Content>>).
+The WIS2 Notification Message allows for transmission of messages in a compact manner, and includes the ability to embedding content inline as required (see <<Properties / Content>>).
 
 include::../requirements/core/REQ_message_size.adoc[]
 
-==== GeoJSON compliance
+//==== GeoJSON compliance
 
-The WIS2 Notification Message schema is based on _GeoJSON_ (RFC7946) and its associated information model.  Compliant messages are therefore compliant with _GeoJSON_.
+//The WIS2 Notification Message schema is based on _GeoJSON_ (RFC7946) and its associated information model.  Compliant messages are therefore compliant with _GeoJSON_.
 
-include::../requirements/core/REQ_validation.adoc[]
+//include::../requirements/core/REQ_validation.adoc[]
 
 ==== Identifier
 
 A universally unique identifier of the message using the UUID standard (RFC4122). The identifier is generated by the originator of the message.
 It provides the anti-loop feature that is needed to ensure that the message will be seen once by all Global Brokers.
 It remains the same throughout the lifetime of the message in the WIS2 ecosystem.
 
-The message identifier is set to a new value by Global Cache services when they publish their modified message for accessing the original core
-data from the cache. The <<data_id>> is retained to ensure traceability and consistency of the same resource.
+//The message identifier is set to a new value by Global Cache services when they publish their modified message for accessing the original core data from the cache. 
+
```

Comment:
I removed this because I think it's external to the spec definition and I believe it's in the guide.